{"files":[{"patch":"@@ -53,0 +53,2 @@\n+import static org.junit.jupiter.api.Assertions.assertTrue;\n+import static org.junit.jupiter.api.Assertions.fail;\n@@ -165,0 +167,1 @@\n+                \/\/ and assert that the cause is a InterruptedException\n@@ -166,3 +169,6 @@\n-                assertThrows(IOException.class, () -> response.body().readAllBytes(), \"expected IOException\");\n-            } catch (Exception e) {\n-                throw new RuntimeException(e);\n+                var thrown = assertThrows(IOException.class, () -> response.body().readAllBytes(), \"expected IOException\");\n+                var cause = thrown.getCause();\n+                assertTrue(cause instanceof InterruptedException, cause + \" is not an InterruptedException\");\n+            } catch (Throwable t) {\n+                t.printStackTrace();\n+                fail();\n","filename":"test\/jdk\/java\/net\/httpclient\/HttpResponseInputStreamInterruptTest.java","additions":9,"deletions":3,"binary":false,"changes":12,"status":"modified"}]}